src/lib/stats.ts

```diff
@@ -0,0 +1,35 @@
+import { CustomClient } from "./client";
+import { Message } from "discord.js";
+import { emojiSuffix } from "interactions/chatInput/emojiSuggest";
+
+const regex = new RegExp(`:[a-zA-Z1-9_]+${emojiSuffix}:`);
```

Comment:
Has this regex been tested to work on `message.content`? I believe emojis in message contents are formatted quite differently (`<:${emojiName}:${emojiId}>`)

Reply:
I'm realizing now that it would match that pattern, so it should be fine.